packages/semantic-ui/src/IconButton/IconButton.js

```diff
@@ -3,8 +3,16 @@ import PropTypes from "prop-types";
 import { Button } from "semantic-ui-react";
 
 function IconButton(props) {
-  const { icon, className, ...otherProps } = props;
-  return <Button icon={icon} className={className} {...otherProps} />;
+  const { icon, iconType, className, ...otherProps } = props;
+  return (
+    <Button
+      icon={icon}
+      size={iconType}
+      tabIndex="-1"
```

Comment:
Let's remove the tabIndex="-1" added here--all buttons in every theme should be accessible

Reply:
Done!